lib/mock_data/default_data_yaml/default_users_and_roles.yml

```diff
@@ -155,8 +155,8 @@ teachers:
 
   teacher_5:
     login: teacher_with_no_class
-    first_name: Teacher_with_no_class
-    last_name: Teacher_with_no_class
+    first_name: TeacherWithNoClass
```

Comment:
Why was this change made?  Is `_` still a valid character for first and last name?